src/tools/distillery.ml

```diff
@@ -1,12 +1,28 @@
 open Printf
 
-(* Constant use for the filename containing the reserve project name for
- * templates
- *)
+(* File containing the reserve project names for templates *)
 let reserve_project_name_filename = ".reserve"
 
+(* File containing files which must be ignored while copying a template. *)
+let eliomignore_filename          = ".eliomignore"
```

Comment:
`.reserve` and `.eliomignore` have two different meanings.
- `.reserve` is for the forbidden projects names, which depends on the template.
- `.eliomignore` is for the files which must be ignored while copying the template.